src/editor.js

```diff
@@ -361,6 +435,14 @@ const EmptyPreview = ({storyDocTemplate}) => html`
   <div class=${n('preview')} data-template=${storyDocTemplate}></div>
 `;
 
+function getBaseUrlPrefix(win) {
```

Comment:
lets move this to a utils file.

Reply:
`src/utils/url.js`?

Reply:
(This might also be no longer needed after #39)